app/components/loading-indicator/template.hbs

```diff
@@ -1,3 +1,7 @@
-<div class='ball-scale ball-light LoadingIndicator'>
+<div class='ball-{{if pulse "pulse" "scale"}} {{if blue "ball-scale-blue"}} {{if dark "ball-dark" "ball-light"}} LoadingIndicator'>
```

Comment:
Drive-by: Per product consultation, remove this, and when no products are found and loading, change the "No projects found" to either be the normal loading pulser or have it say, "Loading…". The pulser is better because a) it should be there already; and b) no extra text.